project/src/main/java/com/google/sps/utilities/DatabaseWrapper.java

```diff
@@ -20,51 +27,58 @@ public DatabaseWrapper(String instanceId, String databaseId) {
   }
 
   /**
-   * Given a user, insert a row with all available fields into the DB
+   * Given a user, insert or update a row with all available fields into the DB
    *
-   * @param user the user to be inserted; user's ID field should not exist in DB
+   * @param user the user to be updated; user's email may or may not exist in DB
    */
-  public void insertUser(User user) {
+  public void insertOrUpdateUser(User user) {
+    // Given a user, update its corresponding row's new fields in DB
     SpannerOptions options = SpannerOptions.newBuilder().build();
     Spanner spanner = options.getService();
     DatabaseId db = DatabaseId.of(options.getProjectId(), instanceId, databaseId);
     DatabaseClient dbClient = spanner.getDatabaseClient(db);
 
-    List<Mutation> mutations = getMutationsFromBuilder(newInsertBuilderFromUser(), user);
+    List<Mutation> mutations = getMutationsFromBuilder(Mutation.newInsertOrUpdateBuilder(USER_TABLE), user);
```

Comment:
I have a similar function for Events -- can you rename 'getMutationsFromBuilder' to 'getUserMutationsFromBuilder'

Reply:
Done